### PR TITLE
added_select-box

### PR DESCRIPTION
**Pull request by murtazshaikh**
review pr..

### CODE REVIEW

File: style.css [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+    box-sizing: border-box;
+}
+body{
+    background: #5f9ea0;
+}
+.box{
+    position: absolute;
+    top: 50%;
+    left: 50%;
+    transform: translate(-50%, -50%);
```
Comment (syedabdullahali) [inline style.css:14]: Explain about transform  and translate why u did use with position absolute

File: style.css [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+    width: 250px;
+    height: 50px;
+    font-size: 20px;
+
+    padding: 10px;
+
+    border: none;
+    outline: none;
+    color: #fff;
+    -webkit-appearance: none;
```
Comment (syedabdullahali) [inline style.css:26]: What is the  use  of -webkit  ?

File: index.html [hunk 3]
```diff
@@ -0,0 +1,24 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="style.css">
```
Comment (syedabdullahali) [inline index.html:7]: What means of rel="stylesheet" ?

**Comment by syedabdullahali** [comment]
Awesome project it will be helpful in your future programming , keep up

**Review by mujeeb100 — APPROVED** [review]
nice..

**Comment by murtazshaikh** [comment]
> Awesome project it will be helpful in your future programming , keep up

thanks @syedabdullahali

**Comment by murtazshaikh** [comment]
> nice..

Thankyou @mujeeb100

**Review by syedabdullahali — APPROVED** [review]

File: index.html (hunk 3 above)
Comment (murtazshaikh) [reply index.html:7]: The 'rel' attribute is used to define the relationship between the linked file and the HTML document.

File: style.css (hunk 2 above)
Comment (murtazshaikh) [reply style.css:26]: Webkit is a web browser rendering engine used by Safari and Chrome, it is used when browser does not support those properties.

File: style.css (hunk 1 above)
Comment (murtazshaikh) [reply style.css:14]: transform is used to translate, rotate or skew some object, here we are using it to center that object

**Pull request merged**